DashDocs/src/chapters/dash_core_components/Graph/examples/figure_graph.jl

```diff
@@ -1,22 +0,0 @@
-using Dash, DashCoreComponents
```

Comment:
Why remove this file?

Reply:
Did not seem to be included anywhere. I can re-add if I was wrong

Reply:
Yes we need this file. It seems to be included too https://github.com/plotly/dash-julia-docs/pull/20/files#diff-0d967af92fd3f1cafcbd1d027fd2340b459bb625a5ec387337b1fa864713aba6R4.

Reply:
Please add it back then I am happy to merge this. Everything else looks good.

Reply:
Thanks -- added back locally. Waiting on v0.1.6 to be released and will rebase this PR off that and push change